Product reviews

mattjio opened the pull request:
this totally works

mattjio committed:
feature files

feature files for reviews and search features

mattjio committed:
Merge branch 'main' of https://github.com/JWS-P3-Team2/Team2-selenium into product-reviews

mattjio committed:
search features and steps

added feature files and steps for search along with review page and additions to home page and main runner

mattjio committed:
reviews

feature files for reviews and steps updated, should be working I hope

cmiller241 committed:
Merge branch 'main' into product-reviews

cmiller241 reviewed (changes requested):
Please change the email tests to test@example.com with 'guest' as a password. And please ensure test cases come back as FAILED or PASS, but doesn't invoke an error.

mattjio committed:
review and search fixes

reviews should show one failure, no timeout, search test cases going through steps

mattjio committed:
Merge branch 'product-reviews' of https://github.com/JWS-P3-Team2/Team2-selenium into product-reviews

cmiller241 committed:
Merge branch 'main' into product-reviews

cmiller241 reviewed (approved):
Looks good. Runs good.
